section11/numberformat_proposed.html

```diff
@@ -621,6 +630,30 @@ <h1>GetNumberFormatPattern ( _numberFormat_, _x_, _exponent_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-getnotationsubpattern" aoid="GetNotationSubPattern">
+      <h1>GetNotationSubPattern ( _numberFormat_, _exponent_ )</h1>
+      <p>
+        The abstract operation GetNotationSubPattern considers the resolved notation and _exponent_, and returns a String value for the notation sub pattern as described in <emu-xref href="#sec-intl.numberformat-internal-slots"></emu-xref>. 
+      </p>
+      <emu-alg>
+        1. Let _localeData_ be %NumberFormat%.[[LocaleData]].
+        1. Let _dataLocale_ be _numberFormat_.[[DataLocale]].
+        1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
+        1. Let _notationSubPatterns_ be _dataLocaleData_.[[notationSubPatterns]].
+        1. Assert: _notationSubPatterns_ is a Record (see <emu-xref href="#sec-intl.numberformat-internal-slots"></emu-xref>).
+        1. Let _notation_ be _numberFormat_.[[Notation]].
+        1. If _notation_ is *"scientific"* or _notation_ is *"engineering"*, then
+          1. Return _notationSubPatterns_.[[scientific]].
+        1. Else if _exponent_ is not 0, then
+          1. Assert: _notation_ is *"compact"*.
+          1. Let _compactDisplay_ be _numberFormat_.[[CompactDisplay]].
+          1. Let _compactPatterns_ be _notationSubPatterns_.[[compact]].[[&gt;_compactDisplay_&lt;]].
+          1. Return _compactPatterns_[[_exponent_]].
```

Comment:
I don't understand the `[[_exponent_]]` part.

Reply:
It should be `.[[_exponent_]]`.  The pattern is dependent on the exponent.